### DISCUSSION TITLE
umi4 怎么在配置 exportStatic 后指定生成目录

### DISCUSSION THREAD

**Discussion by oneduan** [umijs/umi#11681] [Q&A]
需要配置 `exportStatic` 后生成对应路由的文件后指定生成目录， 原文件为目录路由， 放在 `/src/pages` 内， 打包后就直接在 `dist` 了， 查了文档， 没有解决方案， 翻了umijs3 文档找到 `exportStatic.dynamicRoot` ， 但这个api在 umi4中是没有的，umij4 也没有入口去设置配置生成的产物或者路径匹配，  没法配置 ， 求大佬解救 ， 如何配置 ？

**Comment by fz6m** [comment]
如果要部署到非根目录，需要配置 `publicPath` 资源加载位置和 `base` 路由前缀。
